genesyscloud/conversations_messaging_settings/resource_conversations_messaging_settings.go

```diff
@@ -0,0 +1,129 @@
+package conversations_messaging_settings
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"terraform-provider-genesyscloud/genesyscloud/consistency_checker"
+	"terraform-provider-genesyscloud/genesyscloud/provider"
+	resourceExporter "terraform-provider-genesyscloud/genesyscloud/resource_exporter"
+	"terraform-provider-genesyscloud/genesyscloud/util"
+	"terraform-provider-genesyscloud/genesyscloud/util/constants"
+	"terraform-provider-genesyscloud/genesyscloud/util/resourcedata"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mypurecloud/platform-client-sdk-go/v133/platformclientv2"
+)
+
+func getAllAuthConversationsMessagingSettingss(ctx context.Context, clientConfig *platformclientv2.Configuration) (resourceExporter.ResourceIDMetaMap, diag.Diagnostics) {
+	proxy := getConversationsMessagingSettingsProxy(clientConfig)
+	resources := make(resourceExporter.ResourceIDMetaMap)
+
+	messagingSettings, resp, err := proxy.getAllConversationsMessagingSettings(ctx)
+	if err != nil {
+		return nil, util.BuildAPIDiagnosticError(resourceName, fmt.Sprintf("Failed to get Conversations messaging Settings: %s", err), resp)
+	}
+
+	for _, messagingSetting := range *messagingSettings {
+		log.Printf("Dealing with messaging setting id : %s", *messagingSetting.Id)
```

Comment:
No need for this log statement. We found before that it clogs up the logs a good bit when you're running an export. Better to just provide all the details when one fails